bootstrap/kubeadm/kubeadm/v1beta1/utils.go

```diff
@@ -50,7 +50,7 @@ func ConfigurationToYAML(obj runtime.Object) (string, error) {
 // MarshalToYamlForCodecs marshals an object into yaml using the specified codec
 // TODO: Is specifying the gv really needed here?
 // TODO: Can we support json out of the box easily here?
-func MarshalToYamlForCodecs(obj runtime.Object, gv schema.GroupVersion, codecs serializer.CodecFactory) ([]byte, error) {
+func MarshalToYamlForCodecs(obj runtime.Object, gv schema.GroupVersion, codecs serializer.CodecFactory) ([]byte, error) { //nolint
```

Comment:
Why is `nolint` needed here when it wasn't previously?

Reply:
It complains about 
```
 `gv` can be `k8s.io/apimachinery/pkg/runtime.GroupVersioner` (interfacer)
```

Reply:
Was there a way that CABPK avoided that error previously? Especially since it looks like `interfacer` was previously enabled.

Reply:
I'm not sure, but this falls under https://github.com/kubernetes-sigs/cluster-api/issues/1553

also given that these are the kubeadm copied types, it's really low priority to fix

Reply:
I'm good with this falling under #1553 for now